scripts/rename_resources.py

```diff
@@ -19,9 +19,12 @@
 log = logging.getLogger(__name__)
 
 renamed: dict[str, Optional[str]] = {
-    # FIXME: Remove the following entry
-    #        https://github.com/DataBiosphere/azul/pull/6285
-    'opensearch_cluster_settings.index': None
+    # FIXME: Remove the following Macie entries
+    #        https://github.com/DataBiosphere/azul/pull/6322
+    # @formatter:off
+    'aws_securityhub_standards_control.best_practices_macie_1': 'aws_securityhub_standards_control.nist_control_macie_1',  # noqa E501
+    'aws_securityhub_standards_control.best_practices_macie_2': 'aws_securityhub_standards_control.nist_control_macie_2',  # noqa E501
+    # @formatter:on
```

Comment:
Please extract a variable for the string `aws_securityhub_standards_control` and make this a comprehension over `[1,2]`. Let's not have the FIXME. Our convention should be to clean these up when we add new ones after sufficient time has passed.